Library/Formula/cereal.rb

```diff
@@ -16,8 +16,12 @@ class Cereal < Formula
   option "with-tests", "Build and run the test suite"
 
   depends_on "cmake" => :build if build.with? "tests"
+  depends_on :macos => :mavericks
```

Comment:
`ENV.cxx11` in the test won't work as the CFLAGS are never used. Try to compile the test file in this way:

```ruby
system ENV.cc, "-std=c++11", "-stdlib=libc++", "-lc++", "-o", "test", "test.cpp"
```